scripts/checkYarn.js

```diff
@@ -1,4 +1,4 @@
-if (!/yarn\.js$/.test(process.env.npm_execpath || '')) {
+if (process.env.npm_execpath.indexOf('yarn') === -1) {
```

Comment:
Rather something that includes the `js` extension at the end:

```suggestion
if (!/yarn(-[\d.]+)?\.c?js$/.test(process.env.npm_execpath || '')) {
```

It also fallbacks to an empty string

Reply:
ok , more discreet👍